### PR TITLE
fix costumes are unable to be unlocked.

### PR DESCRIPTION
**Pull request by hamusuke0323**
## Description

Please carefully read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md) before making any pull requests.
And, **Do not make a pull request to merge into stable unless it is a hotfix. Use the development branch instead.**
## Type of changes

- [x ] Bug fix
- [ ] New feature 
- [ ] Enhancement
- [ ] Documentation

## Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] My pull request is unique and no other pull requests have been opened for these changes
- [ x] I have read the [Contributing note](https://github.com/Grasscutters/Grasscutter/blob/stable/CONTRIBUTING.md) and [Code of conduct](https://github.com/Grasscutters/Grasscutter/blob/development/CODE_OF_CONDUCT.md)
- [ x] I am responsible for any copyright issues with my code if it occurs in the future.

### CODE REVIEW

**Comment by hamusuke0323** [comment]
We cant get avatar costumes because the property name "costumeId" was changed to "skinId" in AvatarCostumeExcelConfigData.json from V3.1 update probably, I guess.

File: src/main/java/emu/grasscutter/data/excels/AvatarCostumeData.java [hunk 1]
```diff
@@ -6,14 +6,14 @@
 
 @ResourceType(name = "AvatarCostumeExcelConfigData.json")
 public class AvatarCostumeData extends GameResource {
-    private int costumeId;
+    private int skinId;
```
Comment (Birdulon) [inline src/main/java/emu/grasscutter/data/excels/AvatarCostumeData.java:11]:
```suggestion
    @SerializedName(value = "skinId", alternate = "costumeId")
    private int skinId;
```

**Comment by Birdulon** [comment]
Looks like `skinId` is the value from 2.2 NT, we should keep `costumeId` as an alternate name so that old resources don't break for now. (probably remove the alternate name in a patch or two)

**Comment by hamusuke0323** [comment]
I have a unrelated question. May I contribute to gc with using generated .java files from Sorapointa .proto files?

**Comment by Birdulon** [comment]
Yes, that's acceptable

**Comment by hamusuke0323** [comment]
I see. thanks for your reply

**Comment by hamusuke0323** [comment]
sorry, I forgot splitting PR. should I revert?

**Comment by Birdulon** [comment]
Make a new branch at your current `head` (`git branch namecards`) and reset your `development` branch to before those commits (`git reset --hard 9182835`)

**Comment by Birdulon** [comment]
note that you'll also have to force push to `development` (`git push -f`)

**Review by Birdulon — APPROVED** [review]

**Pull request merged**